### DISCUSSION TITLE
A good program for making 3d Modelers

### DISCUSSION THREAD

**Discussion by gnubsdguy** [redeclipse/base#1437] [General]
Hi Hello

I have Red Eclipse and I was wondering, which program do you use to make 3d models for the game ? and I'm asking because i would like to use a program like to that to make my own opensource program.

Thanks ! (Keep the good work with Red Eclipse, amazing game !)

**Comment by SniperGoth** [comment]
Blender, usually, for both modelling/texturing/rigging and animating, exporting etc

**Reply by gnubsdguy** [reply]
Hi an thanks for so fast answer !, yes I have blender installed but do i need some special plugin to use with blender ? (for example MD2 or MD3 files) and thanks again for your fast answer !

**Comment by SniperGoth** [comment]
The formats most used in RE are .obj for static models, and .iqm for animated ones (you'd need a iqm exporter for that one), aside from that, we don't really use any other format.
You can use md3 for very specific things tho, like meshes that are animated via vertex, without any bones (see the "water" at the retrowave race map, that one is md3) but that format is very rarely used if at all.

**Comment by IceflowRE** [comment]
https://github.com/lsalzman/iqm take the iqm blender plugin from here.
